core/src/main/java/org/apache/accumulo/core/metadata/schema/RootTabletMetadata.java

```diff
@@ -59,11 +60,14 @@ public class RootTabletMetadata {
   };
 
   // JSON Mapping Version 1. Released with Accumulo version 2.1.0
-  private static final int VERSION = 1;
+  private static final int VERSION_1 = 1;
+  // JSON Mapping Version 2. Released with Accumulo version 3,1
+  private static final int VERSION_2 = 2;
+  private static final int VERSION = VERSION_2;
 
   // This class is used to serialize and deserialize root tablet metadata using GSon. Any changes to
   // this class must consider persisted data.
-  private static class Data {
+  public static class Data {
```

Comment:
Feel free to ignore this comment.  I would keep this class private

```suggestion
  private static class Data {
```

and add the the following method to `RootTabletMetadata` and have everything call it.

```java
    public static boolean needsUpgrade(final String json) {
      return Data.needsUpgrade(json);
    }
```

Reply:
Addressed in cc6b6e25801c